repose-aggregator/src/docs/asciidoc/filters/regex-rbac.adoc

```diff
@@ -0,0 +1,176 @@
+<3> The resource at `/path/{any-single-segment}/this` can only be accessed with a `POST` if the `X-Roles` header has at least one of the required roles: `role1` or `role2`
+<4> The resource at `/path/{any-single-segment}/this` can only be accessed with a `DELETE` if the `X-Roles` header has the required role: `role1`
+<5> The resource at `/path/{any-single-segment}/that` can only be accessed with a `GET` or `PUT`, but there is no required role that must be in the `X-Roles` header.
+<6> The resource at `/Path/{any-single-segment}` can only be accessed with a `GET` and must either have the `admin` role or both `role1` and `role2` in the `X-Roles` header.
+<7> The resource at `/Path/{any-single-segment}/{anything}` can be accessed with any HTTP method, but must have either `admin` of `role with space` in the `X-Roles` header.
+
+=== Full Example with External Resources File
+This configuration is a full example that uses every possible configuration item.
+It does not, however, cover the resources format, as including resources both inline and via the `href` attribute will cause only the inline resources to be used.
+
+[source,xml]
+.regex-rbac.cfg.xml
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<regex-rbac xmlns="http://docs.openrepose.org/repose/regex-rbac/v1.0"
+             mask-rax-roles-403="false" <!--1-->
+>
+    <delegating <!--2-->
+        quality="0.3" <!--3-->
+        component-name="regex-rbac"/> <!--4-->
+    <resources href="/path/to/resources"/> <!--5-->
+</regex-rbac>
+----
+<1> If set to true, instead of returning a FORBIDDEN (403) or a METHOD NOT ALLOWED (405), the response will be a NOT FOUND (404).
+    (Default: `false`)
+<2> Inclusion of this element prevents this filter from returning errors, and directs this filter to populate delegation headers instead.
+<3> Specifies the quality of specific output headers.
+    When setting up a chain of delegating filters, the highest quality number will be the one that is eventually output.
+    (Default: `0.3`)
+<4> The component name used in the delegation header.
+    This is particularly useful when multiple instances of an API-Checker based filter are used in the same filter chain.
+    (Default: `regex-rbac`)
+<5> Specifies a location to an external file which contains the RegEx RBAC resources.
+    If the message element has a value and the `href` attribute is configured, the RegEx RBAC will use what is configured in the value.
+    If the file that the `href` attribute points to is modified, the RegEx RBAC will not reload the configuration.
+    So the new RBAC file should be placed in a new file name (e.g. Dated) and the regex-rbac.cfg.xml file updated to point to it in order to guarantee the changes are utilized.
+
+== Additional Information
+
+=== Delegation
+In some cases, you may want to delegate the validation of a request down the chain to either another filter or to the origin service.
```

Comment:
`In some cases (User access Events(link to probably non-existent recipe)), ...`

Reply:
Updated.